lab-gavin/README.md

```diff
@@ -0,0 +1,2 @@
+#A great function and sum function written by Gavin Thomas#
+## I hope you enjoy this code, use it freely ##
```

Comment:
Include more details about the project, with an overview of what the functions do. You can also mention testing in your readme.